.github/PULL_REQUEST_TEMPLATE/release_template.md

```diff
@@ -0,0 +1,17 @@
+Link to the milestone on Github e.g. https://github.com/nimblehq/infrastructure-templates/milestone/41?closed=1
```

Comment:
Actually, I think we just need this line as we are using GitHub to manage tasks
```suggestion
https://github.com/nimblehq/infrastructure-templates/milestones/xx
```

Reply:
Updated in https://github.com/nimblehq/infrastructure-templates/pull/21/commits/e1e2e2961b0ad3409eb0b7e969fe9a28b27649fe